modules/lists/list-row.tsx

```diff
@@ -3,12 +3,14 @@ import {PropsWithChildren} from 'react'
 import {Platform, StyleProp, StyleSheet, View, ViewStyle} from 'react-native'
 import {Touchable} from '@frogpond/touchable'
 import {DisclosureArrow} from './disclosure-arrow'
+import * as c from '@frogpond/colors'
 
 const styles = StyleSheet.create({
 	childWrapper: {
 		flex: 1,
 	},
 	container: {
+		backgroundColor: c.secondarySystemGroupedBackground,
```

Comment:
Are we able to remove this one now?

Reply:
Might need more work if we want to remove this. In dark theme, I'm seeing this not work well without this.